src/zenml/zen_server/zen_server_api.py

```diff
@@ -619,7 +975,16 @@ async def pipeline_run(
     responses={409: error_response},
 )
 async def register_pipeline_run(pipeline_run: PipelineRunWrapper) -> None:
-    """Registers a pipeline run."""
+    """Registers a pipeline run.
+
+    # noqa: DAR401
```

Comment:
Is this neccessary?

Reply:
Yes, unfortunately. Haven't found a way to get round it otherwise it throws a failure around what gets raised in this method.